bin/deploy

```diff
@@ -4,13 +4,24 @@
 #
 #   APP defaults to carnival-staging
 #
+#   Set BUILD=1 to perform a Halcyon build
+#
 ###
 set -e
 
 case "${1:-carnival-staging}" in
-  carnival-staging) git push staging master ;;
+  carnival-staging)
+    git push staging master
+
+    if [ "$BUILD" = 1 ]; then
+      heroku run --size=PX build
+      git commit --allow-empty -m 'Trigger deployment'
```

Comment:
So that would mean that all deploys to staging (building or not) will have to force -- right? Is that OK? I get a twitch when automating anything using the word "force"